code/mission/missionmessage.cpp

```diff
@@ -316,7 +316,8 @@ int add_avi( char *avi_name )
 	strcpy_s( extra.name, avi_name );
 	strcpy_s( extra.anim_data.filename, avi_name);
 	extra.num = -1;
-	generic_anim_load(&extra.anim_data);
+	generic_anim_load(&extra.anim_data);   // load only to validate the anim
+	generic_anim_unload(&extra.anim_data); // unload to not waste bmpman slots
```

Comment:
Ah, right. `generic_anim_load` calls `bm_load_animation` and that has several Warnings for bad data. 
(Note that retail ANIs will always fail to load here due to [abc] being appended to the tabled filename to find the real files, so checking the return code would require exceptions).